pyproject.toml

```diff
@@ -35,6 +35,7 @@ dependencies = [
     "Pillow>=9.3.0",
     "pypng>=0.0.20",
     "psutil>=5.9.2",
+    "pygltflib>=1.16.2"

```

Comment:
@randallfrank do we really want this as an hard dependency?
Shouldn't we put this as an hard dependency in the new repository?

Reply:
@mariostieriansys let's reconsider in the context of the potential move to usd-core.

Reply:
Moved to ensight requirements_3_10.txt